### DISCUSSION TITLE
Ability to open a new stream in the right-click menu

### DISCUSSION THREAD

**Discussion by pyth0on** [f1multiviewer/issue-tracker#297] [Feature requests] [answered]
![image](https://user-images.githubusercontent.com/65238629/201514740-a2982385-2ff6-4eac-a2c3-af037c12510d.png)
Additionally to "Switch Stream", it would be useful to add the option "Add Stream". It would basically give you the same list of streams, but opens a new window instead of switching the current stream.

**Comment by f1multiviewer** [comment, marked as answer]
Loved this idea so much, it'll be in the next release. Hold `Alt` while clicking, and it'll open a new one

**Reply by pyth0on** [reply]
Awesome!
